pages/submission-thank-you.tsx

```diff
@@ -1,8 +1,11 @@
 import Layout from "../components/Layout";
 
+const pageDescription =
+  "Add a job opportunity and have it featured on the Jobs Board to get in touch with local talent.";
```

Comment:
"Thank you for submitting to the Code Youngstown Job Board. Please be patient, as we manually approve every submission. We may reach out if we have any questions.";